Makefile

```diff
@@ -57,16 +58,35 @@ e2e: build check-env ginkgo
 ginkgo:
 	GO111MODULE=off go get -u github.com/onsi/ginkgo/ginkgo
 
+unexport USE_DISK_CACHE
+unexport USE_BLOB_CACHE
 ci-e2e:  ## Runs ci e2e tests
 .PHONY: ci-e2e
 # export GITHUB_AUTH_TOKEN with personal access token to run the e2e
 ci-e2e: build check-env
 	$(call ndef, GITHUB_AUTH_TOKEN)
-	mkdir -p $(OUTPUT)
-	mkdir -p cache
-	USE_DISK_CACHE=1 DISK_CACHE_PATH="./cache" ./scorecard --repo=https://github.com/ossf/scorecard --show-details --metadata=openssf  --format json > ./$(OUTPUT)/results.json
-	@sleep 30
-	ginkgo -p  -v -cover --skip="E2E TEST:blob"  ./...
+	@echo Ignoring these test for ci-e2e $(IGNORED_CI_TEST)
+	ginkgo -p  -v -cover --skip=$(IGNORED_CI_TEST)  ./e2e/...
+
+.PHONY: test-disk-cache
+test-disk-cache: build  ## Runs disk cache tests
+	$(call ndef, GITHUB_AUTH_TOKEN)
+	# Start with clean cache
+	rm -rf $(OUTPUT)
+	rm -rf cache
+	mkdir $(OUTPUT)
+	mkdir cache
+	@echo Focusing on these tests $(FOCUS_DISK_TEST)
+	USE_DISK_CACHE=1 DISK_CACHE_PATH="./cache" \
+				   ./scorecard --repo=https://github.com/ossf/scorecard \
+				   --show-details --metadata=openssf  --format json > ./$(OUTPUT)/results.json
+	USE_DISK_CACHE=1 DISK_CACHE_PATH="./cache" ginkgo -p  -v -cover --focus=$(FOCUS_DISK_TEST)  ./e2e/...
+	# Rerun the test with the disk cache
+	USE_DISK_CACHE=1 DISK_CACHE_PATH="./cache" ./scorecard \
```

Comment:
I don't understand the difference from these two commands next from the previous ones. Are previous ones supposed to be run with USE_DISK_CACHE=0 ?

Reply:
Can you also move "./scorecard" to next line, to match other command.

Reply:
The whole idea is to start without a cache in the first run and in the second run it should fetch from the cache. 

The goal of this test is to make sure the cache didn't have any issues.

Reply:
Makes sense, thanks for clarifying this.